src/ui/public/chrome/directives/global_nav/_global_nav.scss

```
@@ -61,10 +61,6 @@
   display: inline-block;
 
   &:hover {
-    background-color: darken($euiColorPrimary, 15%);
-  }
-
-  &.active {
-    background-color: $euiColorPrimary;
+    background-color: darken($euiColorPrimary, 2%);
```

```suggestion
    background-color: darken($kbnChromeGlobalNavBackground, 4%);
```

To match `. kbnGlobalNavLink`?